docs/pages/machine-id/introduction.mdx

```diff
@@ -8,6 +8,16 @@ certificates to any service account (e.g., a CI/CD server) by retrieving
 credentials from the Teleport Auth Service. This enables fine-grained
 role-based access controls and audit.
 
+<iframe
```

Comment:
Is this actually how we embed videos now?

Reply:
It looks like we use different methods for YouTube vs local files. This is the method I found already in use for YT.